src/main/java/io/dropwizard/revolver/http/RevolversHttpHeaders.java

```diff
@@ -34,5 +34,7 @@
     String CALLBACK_TIMEOUT_HEADER = "X-CALLBACK-TIMEOUT";
     String CALLBACK_METHOD_HEADER = "X-CALLBACK-METHOD";
     String CALLBACK_RESPONSE_CODE = "X-RESPONSE-CODE";
+    String RETRY_AFTER = "X-RETRY-AFTER";
+    String DYAMIC_MAILBOX = "X-DYNAMIC-MAILBOX";
```

Comment:
@phaneesh This X-DYNAMIC-MAILBOX will be used to evaluate whether to move to dynamic mailbox or not. Would help us maintain backward compatibility